controllers/account.js

```diff
@@ -116,4 +116,17 @@ router.post('/preferences', (req, res, next) => {
 		.catch(() => 'Es ist ein Fehler bei den Präferenzen aufgetreten!');
 });
 
+router.post('/ttl', (req, res, next) => {
+	if (req.body.resetTimer) {
+		api(req).post('/accounts/jwtTimer')
```

Comment:
```suggestion
		return api(req).post('/accounts/jwtTimer')
```

Reply:
not realy necessery or what should be the benefit?

Reply:
https://stackoverflow.com/a/43055671 not really not necessary (?!?) means there should be a return! https://www.reddit.com/r/node/comments/7p2e0o/best_practice_to_return_on_send_in_express/

Reply:
Ce careful from a promise return by intern calls, is not the same like undefined. 
I the else is removed then it must be add a return to aviod the code behind. 
But it is a huge logic change for intern calls. I do not know, it it is used for intern calls.